src/hal/src/pso/graphics.rs

```diff
@@ -104,15 +121,16 @@ pub enum FrontFace {
     CounterClockwise,
 }
 
-///
+/// A depth bias allows you to specify a hint for how to draw
```

Comment:
I'd say "A depth bias allows changing the produced depth values for fragments slightly but consistently, allowing to draw multiple layers of the same geometry without Z-fighting"